packages/libxml2_autotools.rb

```diff
@@ -0,0 +1,46 @@
+Package.load_package("#{__dir__}/libxml2.rb")
+# cmake builds are dependent upon libxml2, so we need a libxml2 build
```

Comment:
In that case, why are we adding another version and not just using autotools on the main build?

Reply:
A bunch of cmake specific files get added to the build if we use cmake, which I guess are used by other builds...

Reply:
```
crew files libxml2 | grep cmake
/usr/local/lib64/cmake/libxml2-2.13.3/libxml2-config-version.cmake
/usr/local/lib64/cmake/libxml2-2.13.3/libxml2-config.cmake
/usr/local/lib64/cmake/libxml2-2.13.3/libxml2-export-release.cmake
/usr/local/lib64/cmake/libxml2-2.13.3/libxml2-export.cmake
```

Reply:
But cmake needs libxml2 to build and circular dependencies don't work with libraries with versioned symbols... great.

Reply:
Although-- https://gitlab.gnome.org/GNOME/libxml2/-/issues/709 and https://gitlab.gnome.org/GNOME/libxml2/-/issues/708 seem to suggest we might be able to get away with not using icu?